src/_blog/cryptotls.md

```diff
@@ -0,0 +1,60 @@
+---
+tags:
+- quic
+- go
+- standard library
+- crypto/tls
+title: A QUIC API for crypto/tls
+description:
+date: 2023-08-15
+permalink: "/2023-08-15-quic-crypto-tls/"
+translationKey: ''
+header_image: /quic-crypto-tls.png
+author: Marten Seemann
+---
+
+# A QUIC API for crypto/tls
```

Comment:
```suggestion
# A QUIC API for crypto/tls
```
Should we make it more clear this is in Go?

Reply:
I've updated the title. Suggestions for a better one welcome.